packages/path_provider/path_provider_foundation/lib/path_provider_foundation.dart

```diff
@@ -68,4 +68,13 @@ class PathProviderFoundation extends PathProviderPlatform {
   Future<String?> getDownloadsPath() {
     return _pathProvider.getDirectoryPath(DirectoryType.downloads);
   }
+
+  /// Returns the path to the container of the specified App Group.
+  Future<String?> getContainerPath({required String appGroupIdentifier}) async {
+    if (!Platform.isIOS) {
```

Comment:
Is there a reason to not support this on macOS?

Reply:
App group identifiers are a bit different on macOS: https://developer.apple.com/documentation/xcode/configuring-app-groups. You use the team identifier instead of a unique string. So I figured it would make more sense to think through validation and testing for that use case. I didn't prioritize it because I don't think App Groups with macOS apps are super relevant for our users right now. We can add it in the future if it comes up though!

Reply:
Created https://github.com/flutter/flutter/issues/120828 to track implementing with macOS.

Reply:
Perfect thanks!